app/src/main/res/values-it/strings.xml

```diff
@@ -227,8 +228,32 @@
     <string name="or_character">O</string>
     <string name="mark_notification_title">Disabilita lo stato di notifica letta</string>
     <string name="mark_notification_summary">Premendo su una notifica non la segnerà come letta.</string>
+    <string name="light_theme_mode" translatable="false">Tema chiaro</string>
+    <string name="dark_theme_mode" translatable="false">Tema scuro</string>
+    <string name="red_theme_mode" translatable="false">Red</string>
+    <string name="pink_theme_mode" translatable="false">Pink</string>
+    <string name="purple_theme_mode" translatable="false">Purple</string>
+    <string name="deep_purple_theme_mode" translatable="false">Deep Purple</string>
+    <string name="indigo_theme_mode" translatable="false">Indigo</string>
+    <string name="blue_theme_mode" translatable="false">Blue</string>
+    <string name="light_blue_theme_mode" translatable="false">Light Blue</string>
+    <string name="cyan_theme_mode" translatable="false">Cyan</string>
+    <string name="teal_theme_mode" translatable="false">Teal</string>
+    <string name="green_theme_mode" translatable="false">Green</string>
+    <string name="light_green_theme_mode" translatable="false">Light Green</string>
+    <string name="lime_theme_mode" translatable="false">Lime</string>
+    <string name="yellow_theme_mode" translatable="false">Yellow</string>
+    <string name="amber_theme_mode" translatable="false">Amber</string>
+    <string name="orange_theme_mode" translatable="false">Orange</string>
+    <string name="deep_orange_theme_mode" translatable="false">Deep Orange</string>
     <string name="theme_title">Scegli un tema</string>
     <string name="theme_summary">Scegli il tuo tema predefinito</string>
+    <string name="theme_color_summary">Scegli il colore di evidenza</string>
+    <string name="theme_color_title">Colore evidenziatore</string>
+    <string name="logo_designer" formatted="true" translatable="false">Logo Designer (%s)</string>
+    <string name="google_plus" translatable="false">Google+</string>
```

Comment:
This is done but the website sting is there twice please remove one of them

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_